src/components/core/ApplicationHeader.tsx

```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="relative cursor-pointer">
+							<Link href="https://forms.office.com/r/z48ExG8dPs">
```

Comment:
Probably better to make this open a new tab, pass the target attribute accordingly